.github/workflows/common-flow-host.yml

```diff
@@ -10,6 +10,16 @@ on:
         type: string
       get-redis:
         type: boolean
+      test-config:
+        description: 'Test configuration environment variable (e.g. "CONFIG=concurrent_write" or "QUICK=1")'
+        required: true
+        type: string
+      coordinator:
+        type: boolean
+        default: true
+      standalone:
+        type: boolean
+        default: true
 
 env:
```

Comment:
There are some differences, this values are not suppose to be inputs so it better to define them here. What do you mean by “where it is required”?

Reply:
Where does the `sudo` required?

Reply:
oh. it is probably not used right now, but some environments require `sudo` while others don't recognize the command so it will be needed when we remove readies